src/main/java/com/cgm/infolab/controller/UserApiController.java

```diff
@@ -25,10 +28,26 @@ public UserApiController(UserRepository userRepository) {
     }
 
     @GetMapping("/api/users")
-    public List<UserDto> getUsername(@RequestParam("user") String user) {
+    public List<UserDto> getUsername(@RequestParam("user") String user,
+                                     @RequestParam(required = false, name = "page[size]") Integer pageSize,
+                                     @RequestParam(required = false, name = "page[before]") String pageBefore,
+                                     @RequestParam(required = false, name = "page[after]") String pageAfter) {
+
+        if (pageBefore != null && pageAfter != null) {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Range Pagination Not Supported");
```

Comment:
`"Range Pagination Not Supported"`. Si un po' più esplicito. Tipo `"Range Pagination Not Supported. Query parameters page[before] and page[after] can't be used together"`

Visto che il messaggio è un po' lungo metterei tutto, compresa la verifica in un helper. Esempio

`ApiHelper.throwOnRangePagination()`

Così poi lo usi anche nelle altra API